_maps/map_files/YogStation/YogStation.dmm

```diff
@@ -28859,6 +28859,10 @@
 	pixel_x = -24
 	},
 /obj/item/folder/yellow,
+/obj/item/storage/firstaid/regular{
```

Comment:
This is the real important addition of this PR. I missed you so much cargo medkit

Reply:
it's already there, I just moved it.